src/goto-symex/renaming_level.h

```diff
@@ -18,34 +18,41 @@ Author: Romain Brenguier, romain.brenguier@diffblue.com
 #include <util/irep.h>
 #include <util/ssa_expr.h>
 
+/// Wrapper for a \c current_names map, which maps each identifier to an SSA
```

Comment:
:question: What do we use this map for? Could you explain a bit more about the context of this?